### PR TITLE
Minior cleanup in pkg/controller/node/ipam/sync/sync.go

### PR DESCRIPTION
**Pull request by jingax10**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controller/node/ipam/sync/sync.go [hunk 1]
```diff
@@ -173,7 +173,7 @@ func (sync *NodeSync) Delete(node *v1.Node) {
 
 // syncOp is the interface for generic sync operation.
 type syncOp interface {
-	// run the requested sync operation.
+	// Run the requested sync operation.
```
Comment (bowei) [inline pkg/controller/node/ipam/sync/sync.go]: actually go want this to be lowercase `run`

Comment (jingax10) [reply pkg/controller/node/ipam/sync/sync.go]: Thanks! Reverted the change. PTAL

**Comment by cblecker** [comment]
/lgtm
/assign @bowei 
for approval

**Pull request merged**